addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -97,6 +97,10 @@ class StudyService extends Service {
     // For now, we assume that 'createdBy' and 'updatedBy' are always users and not groups
     const by = _.get(requestContext, 'principalIdentifier.uid');
 
+    if (rawData.category === 'Open Data' && by !== '_system_') {
+      throw this.boom.badRequest('Only the system can create Open Data studies.');
```

Comment:
Please add true so that the message can be propagated to across boundaries, otherwise, the UI will just see 'something went wrong'.
throw this.boom.badRequest('Only the system can create Open Data studies.', true);

Reply:
Done